firefox_accounts/firefox_accounts.model.lkml

```diff
@@ -1,7 +1,23 @@
 connection: "telemetry"
 label: "Firefox Accounts"
+include: "//looker-hub/firefox_accounts/views/fxa_first_seen_table.view.lkml"
 include: "//looker-hub/firefox_accounts/explores/*"
-include: "//looker-hub/firefox_accounts/dashboards/*"
 include: "views/*"
```

Comment:
This made my head spin a bit but I think I get it now.

So having this include statement basically forces the extensions of the explores below to use the extended versions of the views (defined in e.g. `firefox_accounts/views/events.view.lkml`). This is what essentially overrides the implicit reliance of the imported explores on the original (un-extended) version of the views.

Reply:
Right, you got it. Some explanation here: https://docs.looker.com/data-modeling/learning-lookml/refinements#refinements_are_applied_in_order.